src/Redux/tracker/tracker.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable */
```

Comment:
- [ ] Linters help us write better code. Let's maintain that. Kindly remove eslint disabling here and correct the resulting error(s).